objectivec/GPBBootstrap.h

```diff
@@ -95,7 +95,11 @@
  * Attribute used for Objective-C proto interface deprecations.
  **/
 #ifndef GPB_DEPRECATED
-#define GPB_DEPRECATED __attribute__((deprecated))
+#if __has_extension(attribute_deprecated_with_message)
+#define GPB_DEPRECATED(...) __attribute__((deprecated("" __VA_ARGS__)))
```

Comment:
Why does this need to be varargs vs just take a single _msg_?

Reply:
I guess the catch would be if we ever wanted to support `[[gnu::deprecated]]` or `[[deprecated]]`.  Given it now is a `_MSG` form, lets go with one arg so no one were to think it was a printf style args…

Reply:
Done.